### ISSUE TITLE
Wrong hiscore config?

### ISSUE THREAD

**Issue by tacertain** [MiSTer-devel/Arcade-Bagman_MiSTer#24]
https://github.com/MiSTer-devel/Arcade-Bagman_MiSTer/blob/215be747fee10e8555d2f8b4272b7aa62f089a2b/Arcade-Bagman.sv#L422

Hey @JimmyStones, I put the wrong value here, didn't I? It only needs to be a width of 3 to handle the 5 rows of config in botanic, right? I don't need you to fix - just checking my understanding. If I'm correct, I'll fix. If I'm not correct, please enlighten me!

**Comment by JimmyStones** [comment]
You are correct - CFG_ADDRESSWIDTH(3) would give you support for 8 rows (0-7).   Does it absolutely no harm having more capacity than necessary though, just wastes a little bit of block ram.
